Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/MobilityCommonBridge.java

```diff
@@ -774,6 +845,19 @@ public void run() {
                             locateOnMapManager.setMarkerCallback(markerCallback, markerCallbackForTags);
                             locateOnMapManager.setEnableMapClickListener(payload.optBoolean("enableMapClickListener", false));
 
+                            if (googleMap != null && EDIT_LOCATION) {
+                                LatLng source = new LatLng(Double.parseDouble(lat), Double.parseDouble(lon));
+                                if (googleMap != null) {
```

Comment:
this check can be removed from here as already this check is added in line no 848.